includes/Webhook/Service/WebhookSubscriptionService.php

```diff
@@ -205,6 +205,9 @@ private function getExistingWebhook(AbstractWebhookWebService $service, string $
             return null;
         }
 
+        return $existingWebhook;
+
+        // TODO the validation could cause a loop (10 jan 2022), investigate and fix
```

Comment:
create a ticket for this and comment below code instead of leaving it unreachable

Reply:
When you have a ticket use the number in the command